recipes/frogs/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ls -l
+cp -r ./* $PREFIX/
+mkdir $PREFIX/bin
+cd $PREFIX/app
+ln -s ./* ../bin/
```

Comment:
Looks like a lot of XML and HTML in `app`, do these need to be in `bin`?

Also, copying everything into `$PREFIX` doesn't follow the FHS (https://bioconda.github.io/guidelines.html#filesystem-hierarchy-standard)